package.json

```diff
@@ -9,7 +9,7 @@
     "start": "rimraf dist && eleventy --serve",
     "check-broken-links": "npm run check-broken-links:internal && npm run check-broken-links:external",
     "check-broken-links:internal": "npx broken-link-checker-local ./dist -roe --filter-level 3 --exclude \"*/__/firebase/*\"",
-    "check-broken-links:external": "npx broken-link-checker-local ./dist -roi --filter-level 3 --exclude \"*/__/firebase/*\" --exclude \"*/stenciljs.com/*\" --exclude \"*/twitter.com/*\" --exclude \"*/gramsbergen.nl/*\" --exclude \"*/bookmydesk.com/\" --exclude \"*/play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral*\" --exclude \"*/phaser.io*\" --exclude \"*/www.cameronvetter.com/*\""
+    "check-broken-links:external": "npx broken-link-checker-local ./dist -roi --filter-level 3 --exclude \"*/__/firebase/*\" --exclude \"*/stenciljs.com/*\" --exclude \"*/twitter.com/*\" --exclude \"*/gramsbergen.nl/*\" --exclude \"*/bookmydesk.com/\" --exclude \"*/play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral*\" --exclude \"*/phaser.io*\" --exclude \"https://phaser.io/\""
```

Comment:
_(Already merged, but commenting)_
@mschilling, didn't `--exclude \"*/phaser.io*\"` cover it enough, that you also added `-exclude \"https://phaser.io/\"`? Or should the other one be adjusted instead?